packages/grid/_modules_/grid/GridComponent.tsx

```diff
@@ -111,12 +111,17 @@ export const GridComponent = React.forwardRef<HTMLDivElement, GridComponentProps
 
     const showNoRowsOverlay = !props.loading && totalRowCount === 0;
     const showNoResultsOverlay = !props.loading && totalRowCount > 0 && visibleRowCount === 0;
+
+    const ariaProps = {
+      'aria-label': props['aria-label'],
+    };
```

Comment:
AFAIK, it's also important to add:
```suggestion
      'aria-labelledby': props['aria-labelledby'],
    };
```

for instance: https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html

Reply:
I thought about it. AFAIK, we need a corresponding `id` for `aria-labelledby` where the id refers to the element which has a text needed for labelling. Do we have `header` or any element where we show static text?

Reply:
It's up to the developers to provide, my point is that we currently prevent them to do it.

Reply:
I added `aria-labelledby` on the root.